enterprise_access/apps/api/serializers/content_assignments/assignment.py

```diff
@@ -258,3 +307,31 @@ def get_content_metadata(self, obj):
         if metadata_lookup and (assignment_content_metadata := metadata_lookup.get(obj.content_key)):
             return ContentMetadataForAssignmentSerializer(assignment_content_metadata).data
         return None
+
+
+class LearnerContentAssignmentWithLearnerAcknowledgedResponseSerializer(
+    LearnerContentAssignmentWithContentMetadataResponseSerializer
+):
+    """
+    Read-only serializer for LearnerContentAssignment records that also includes whether or not the learner has
+    acknowledged the assignment.
+    """
+    learner_acknowledged = serializers.SerializerMethodField(
+        help_text='Whether or not the learner has acknowledged the assignment.',
```

Comment:
nit: is this really "the learner has acknowledged expiration or cancelation of the assignment"?

Reply:
+1